O365.Security.Native.ETW.Debug.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
     <metadata>
         <id>Microsoft.O365.Security.Native.ETW.Debug</id>
-        <version>4.1.24</version>
+        <version>4.1.25</version>
```

Comment:
Should we consider a major version bump or at least a minor version bump? This change the behavior from throwing an exception on the trace to only calling the callback when this type of error occurs. Someone could update krabs and now fail silently. We should also clearly indicate the change on the release note and explain what the consumer needs to change when updating to this version.

Reply:
Good call! I bumped the version from 4.1.24 to 4.2.0 and improved the release note.